build/source/driver/multi_driver.f90

```diff
@@ -325,10 +481,10 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 allocate(upArea(nHRU),stat=err); call handle_err(err,'problem allocating space for upArea')
 
 ! identify the total basin area (m2)
-totalArea => bvar_data%var(iLookBVAR%basin__totalArea)%dat(1)
+totalArea = bvarStruct%var(iLookBVAR%basin__totalArea)%dat(1)
```

Comment:
This does not do anything since in the next line it is overwritten by `0._dp`. Should the next line be removed?